src/main/java/org/apache/fineract/operations/TransactionRequest.java

```diff
@@ -80,6 +80,9 @@ public class TransactionRequest extends AbstractPersistableCustom<Long> {
     @Column(name = "CLIENTCORRELATIONID")
     private String clientCorrelationId;
 
+    @Column(name = "error_information")
```

Comment:
let's follow the same variable naming convention as already available in the same class. ie, upper case column name